src/main/java/com/gotocompany/firehose/config/ObjectStorageServiceConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.gotocompany.firehose.config;
+
+import org.aeonbits.owner.Config;
+
+public interface ObjectStorageServiceConfig extends Config {
```

Comment:
Should we call DlqObjectStorageServiceConfig ??

Reply:
This one is a generic config class that can be used for actual blob sink and dlq. Similar to `S3Config` and `GCSConfig` mas. So we should not prepend Dlq as the prefix

Reply:
Blob Sink and Dlq Blob Storage use the same config class, here we can see the key are using placeholder `${OSS_TYPE}` in which, the value could be either `BLOB_STORAGE` or `DLQ`, indicating the configuration to be parsed based on the context